widgets/controls.pas

```diff
@@ -1385,6 +1398,7 @@ procedure TControl.SetWidth(AValue: NativeInt);
 
 procedure TControl.Click;
 begin
+  Writeln('Click = ', Name);
```

Comment:
Please don't include debug statements in the final commits, they'll only be forgotten sooner or later 😅

Reply:
Thanks.